modules/friendlytag.js

```diff
@@ -907,6 +975,14 @@ Twinkle.tag.file.replacementList = [
 	{ label: '{{PNG version available}}', value: 'PNG version available' },
 	{ label: '{{Vector version available}}', value: 'Vector version available' }
 ];
+Twinkle.tag.file.replacementList.forEach(function(idx,el) {
```

Comment:
It can get maddening, but jQuery's `.each` uses `index, value` while javascript's `forEach` uses `value, index`.

```suggestion
Twinkle.tag.file.replacementList.forEach(function(el,idx) {
```